packages/nft-analytics/test/unit/controllers.test.ts

```diff
@@ -12,7 +12,7 @@ describe('controllers', () => {
 
   process.env.API_KEY = 'fake-api-key'
   process.env.LOG_LEVEL = 'debug'
-  process.env.API_PROVIDER = ProviderName.RARIFY
+  process.env.API_PROVIDER = ProviderName.RARIFY // API_PROVIDER is not relevant for these tests
```

Comment:
Great comment
Although I've just realised that none of the env vars are required for these tests, so they can be removed. My bad for not having them removed on the Rarify work.